play/plugin/src/test/kotlin/com/github/triplet/gradle/play/tasks/shared/PublishArtifactIntegrationTests.kt

```diff
@@ -63,7 +63,7 @@ interface PublishArtifactIntegrationTests : SharedIntegrationTest {
         // language=gradle
         val config2 = """
             android.defaultConfig.versionCode 2
-            play.commit = false
+            play.commit = true
```

Comment:
I'm looking at this again and I'm not sure it's right. Does it still fail if you go back to false?

Reply:
The test now passes with both values, so I misinterpreted the test's meaning. Reverted

Reply:
Perfect!